index.html

```diff
@@ -0,0 +1,159 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>☯️ Conference </title>
+    <link rel="stylesheet" href="style.css" />
+    <link
+      rel="stylesheet"
+      href="https://pro.fontawesome.com/releases/v5.10.0/css/all.css"
+      integrity="sha384-AYmEC3Yw5cVb3ZcuHtOA93w35dYTsvhLPVnYs9eStHfGJvOvKxVfELGroGkvsg+p"
+      crossorigin="anonymous"
+    />
+  </head>
+  <body>
+    <section class="grid-container">
+      <section class="Section-1">
+        <div class="social-icons">
+          <a href="#"><i class="fab fa-twitter"></i></a>
+          <a href="#"><i class="fab fa-facebook"></i></a>
+          <ul class="social-lists">
+            <li><a href="#">English</a></li>
+            <li><a href="#">MyPage</a></li>
+            <li><a href="#">Logout</a></li>
+          </ul>
+        </div>
+      </section>
+      <section class="Section-2">
+        <div class="header">
+          <img src="images/logo.png" alt="" class="logo" />
+          <nav class="nav-bar">
+            <ul class="nav-menu">
+              <li class="nav-item">
+                <a href="About.html" class="nav-link">About</a>
+              </li>
+              <li class="nav-item">
+                <a href="#" class="nav-link">Program</a>
+              </li>
+              <li class="nav-item">
+                <a href="#" class="nav-link">Join</a>
+              </li>
+              <li class="nav-item">
+                <a href="#" class="nav-link">Sponsor</a>
+              </li>
+              <li class="nav-item">
+                <a href="#" class="nav-link">News</a>
+              </li>
+              <li><a href="#" class="contact">Contact us</a></li>
+            </ul>
+            <div class="hamburger">
+              <span class="bar"></span>
+              <span class="bar"></span>
+              <span class="bar"></span>
+            </div>
+          </nav>
+        </div>
+      </section>
+
+      <section class="Section-3">
+        <div class="Content">
+          <h1>
+            2nd Global Submit on Applied Science, Engineering and Technology
+            (GSASET)2023
+          </h1>
+          <div class="infomation">
+            <p>
+              The three days of the conference will contribute to build
+              long-term profitable and sustainable communication among the
+              participants in a wide variety of scientific areas with the common
+              interest in Applied Science, Engineering and Technology.
+              </p>
+            </div>
+            <div class="under-content">
+            </p>
+            <h2>2023.03.23(TUE) ~ 25(WED)</h2>
+            <p>&copy SHAGRI-LA HOTEL, BANGKOK, THAILAND</p>
+          </div>
+          
+        </div>
+      </section>
+    </section>
```

Comment:
Kindly ensure that your design matches that of the actual design. Check the margin, paddings, fonts etc. The images below are just limited examples. Kindly go through all your project and ensure that it conforms to the design as much as possible

Your design            |  The design template
:-------------------------:|:-------------------------:
![Screenshot from 2022-10-27 13-47-09](https://user-images.githubusercontent.com/65233860/198302451-3851d451-fcbe-4600-897b-99bc58b3b1f1.png)  |  ![Screenshot from 2022-10-27 13-48-11](https://user-images.githubusercontent.com/65233860/198302517-7ffe329e-c01d-4b75-9e35-628221f113cc.png)


Your design            |  The design template
:-------------------------:|:-------------------------:
![Screenshot from 2022-10-27 13-47-21](https://user-images.githubusercontent.com/65233860/198302637-65d0ee44-b9d1-4524-ab65-d6b856c17632.png)  |  ![Screenshot from 2022-10-27 13-48-19](https://user-images.githubusercontent.com/65233860/198302668-f30ea629-a08e-4bea-92c8-fc2e637bc43c.png)